qa/suites/fs/functional/tasks/client-recovery.yaml

```diff
@@ -8,6 +8,9 @@ overrides:
       - slow request
       - MDS_CLIENT_LATE_RELEASE
       - t responding to mclientcaps
+      - Degraded data redundancy
+      - \(MDS_CLIENTS_LAGGY\)
```

Comment:
Wouldn't just using `MDS_CLIENTS_LAGGY` be enough (i.e., without `()`)? Maybe this was causing one of the failed tests even when the warning is added to ignore-list?

Reply:
Yeah this caused failure because somehow parenthesis is not escaped in grep in ubuntu. `MDS_CLIENTS_LAGGY` can be used, i was just trying to follow with the existing approach seen in teuthology log `egrep -v '\(MDS_ALL_DOWN\)' | egrep -v '\(MDS_UP_LESS_THAN_MAX\)' ....`

Reply:
done